src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -304,21 +304,26 @@ export class BeneficiariesService {
 					block
 					mobile
 					dob
-				    program_beneficiaries {
-					id
-					facilitator_id
-					status
-					enrollment_date
-					
-				  }
+					program_beneficiaries {
+						id
+						facilitator_id
+						status
+						enrollment_date
+						facilitator_id
+						facilitator_user{
```

Comment:
if this is relation - add comment for it at the top

Reply:
added comment